src/main/java/org/kcctl/command/GetPluginsCommand.java

```diff
@@ -33,13 +37,19 @@
 import picocli.CommandLine.Command;
 
 @Command(name = "plugins", description = "Displays information about available connector plug-ins")
-public class GetPluginsCommand implements Runnable {
+public class GetPluginsCommand implements Callable<Integer> {
+
+    private final Version requiredVersionForAllPlugins = new Version(3, 2);
+    private final Set<String> defaultPlugins = Set.of("source", "sink");
```

Comment:
How about getting all plug-in types by default? I think the backwards compatibility concern is less important for kcctl given its current pre 1.0 state.

Reply:
Good idea, updated!